src/output-json.h

```diff
@@ -50,6 +50,8 @@ typedef struct JsonAddrInfo_ {
     Port sp;
     Port dp;
     char proto[JSON_PROTO_LEN];
+    // With this flag, ports are logged only if they have meaningful values.
```

Comment:
comment suggests the value of ports is evaluated, but this is not what I'm seeing?